ci-operator/config/openstack-k8s-operators/manila-operator/openstack-k8s-operators-manila-operator-main.yaml

```diff
@@ -69,6 +73,19 @@ tests:
     pre-commit run -a --show-diff-on-failure
   container:
     from: openstack-operator-ci-pre-commit
+- as: manila-operator-build-deploy-tempest
+  cluster_claim:
+    architecture: amd64
+    cloud: vexxhost
+    owner: openstack-k8s-operators
+    product: ocp
+    timeout: 1h20m0s
+    version: "4.10"
+  optional: true
+  steps:
+    env:
+      TEMPEST_REGEX: manila_tempest_tests.tests.api
```

Comment:
@gouthampacha @cescgina is the `API` regex enough to trigger the required tests? I think we should skip those related to the mount of the share on the running workload because we don't have a compute node in prow jobs, but I'm wondering if we can extend it a bit to improve coverage.

Reply:
personally I have no strong opinion, as I have no experience with manila, I will agree with what you (from the manila team) decide

Reply:
Hi o/, thanks for checking! tests in the manila-tempest-plugin repository are in three buckets: `api`, `scenario` and `rbac`... "api" tests would involve everything except mounting shares; "scenario" tests involve mounting shares and we'd need a nova compute node to create client VMs. The tests would create these to mount shares and do some r/w. "rbac" tests may eventually be interesting here... but they're still being developed upstream... we can enable them once upstream's done with that effort (targeted during the 2023.2/bobcat release)